python/lsst/rubintv/production/slac/botTesting.py

```diff
@@ -393,19 +397,22 @@ def writeRebHeaderShard(self, expRecord, raw):
 
         if self.instrument == 'LSST-TS8':
             detectorList = TS8_REB_HEADER_DETECTORS
+            rebNumber = detector.getName().split('_')[1][1]  # This is the X part of the S part of R12_SXY
+            itemName = f'REB{rebNumber} Header'
         elif self.instrument == 'LSSTComCam':
```

Comment:
Why not merge this into one `if self.instrument == 'LSST-TS8' or self.instrument == 'LSSTCam':` statement, if they're identical?

Reply:
They're actually not quite identical - different detector lists for the different instruments. Going to merge this as-is here, but just FYI that all this stuff will get ripped out and done properly in DM-39922